scientific_library/tvb/simulator/monitors.py

```diff
@@ -371,8 +368,7 @@ class Projection(Monitor):
 
     region_mapping = Attr(
         RegionMapping,
-        required=False,
-        label="region mapping",  #order=3,
+        label="Region Mapping",  #order=3,
```

Comment:
Comments like `#order=...` can be removed (they are deprecated)

Reply:
Done.